lib/generate.js

```diff
@@ -188,7 +189,11 @@ function generateLayoutInternal(options, callback) {
                     if (item.stretchY) { obj[itemIdToUpdate].stretchY = item.stretchY; }
                 });
             });
-            return callback(null, obj);
+            return callback(null, obj, {
+                width: sprite.w,
+                height: sprite.h,
+                items: imagesWithSizes
+            });
```

Comment:
Since this object is created regardless now, maybe we can define it once and refer to it in both places? Also, maybe we could rename `obj` here to `dataLayout` and then have a clear `return callback(null, dataLayout, imgLayout)`

Reply:
Also, is there anything we can do in the function comments (lines 19-32) to describe this change?

Reply:
Great suggestions. Addressed them all in 5019c9a, hopefully.